bin/cml-runner.js

```diff
@@ -17,29 +17,31 @@ const {
 
   RUNNER_PATH = `${WORKDIR_BASE}/${NAME}`,
   RUNNER_IDLE_TIMEOUT = 5 * 60,
-  RUNNER_DESTROY_DELAY = 30,
+  RUNNER_DESTROY_DELAY = 10,
```

Comment:
any reason for this change?

Reply:
30 seconds its too much for what we need. We started setting 30 seconds to try and I think that 10 seconds is far better time. This is just to give the SSH connection at TPI time to display the error.

Reply:
Given that 10 seconds is the maximum exponential backoff polling interval for the provider, I'd say that it wouldn't hurt to have at least a 20 second timeout to stay above the nyquist rate, lest missing the error message if the machine gets terminated sooner.

Reply:
```go
return utils.RunCommand("journalctl --no-pager", 10*time.Second,
```

Or reduce it to 5 I find 10 seconds terribly suboptimal taking into account that we do it to check the healthiness.

Reply:
Opened a ticket in TFI